### PR TITLE
Fix ticket read no right message

### PR DESCRIPTION
**Pull request by Lainow**
Issue : !34524

Fixed redirection to the ticket database page if you do not have access rights.

### CODE REVIEW

File: front/ticket.form.php [hunk 1]
```diff
@@ -88,4 +89,17 @@
...
 
-Html::back();
+if ($track->can($_POST["tickets_id"], READ)) {
+    $toadd = '';
+   // Copy solution to KB redirect to KB
+    if (isset($_POST['_sol_to_kb']) && $_POST['_sol_to_kb']) {
+        $toadd = "&_sol_to_kb=1";
+    }
+    Html::redirect(Ticket::getFormURLWithID($_POST["tickets_id"]) . $toadd);
+}
```
Comment (stonebuzz) [inline front/ticket.form.php:99]: is really needed ?

**Comment by stonebuzz** [comment]
can you adapt `CHANGELOG.md`

File: CHANGELOG.md [hunk 2]
```diff
@@ -7,6 +7,10 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 
 ## [unrelease] -
 
+### Fixed
+
+- Correction of the read no right message ticket and redirection
```
Comment (stonebuzz) [inline CHANGELOG.md:12]:
```suggestion
- Redirect user after escalation if he has no rights to the ticket
```

**Review by stonebuzz — CHANGES_REQUESTED** [review]

**Review by Rom1-B — APPROVED** [review]

**Review by stonebuzz — APPROVED** [review]

**Comment by stonebuzz** [comment]
waiting for customer feedback

**Comment by Lainow** [comment]
> waiting for customer feedback

!34524
The ticket has been resolved because the customer has not replied despite 3 reminders.

**Comment by stonebuzz** [comment]
Hi @Lainow 

can you rebase (We are going to merge)

++

File: CHANGELOG.md [hunk 3]
```diff
@@ -7,6 +7,10 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 
 ## [unrelease] -
 
+### Fixed
+
+- Redirect user after escalation if he has no rights to the ticket
```
Comment (stonebuzz) [inline CHANGELOG.md:12]:
```suggestion
- Redirect users without ticket rights after escalation.
```

**Review by stonebuzz — CHANGES_REQUESTED** [review]

**Review by stonebuzz — APPROVED** [review]

**Pull request merged**